assets/js/controllers/datepicker_controller.test.js

```diff
@@ -0,0 +1,176 @@
+/*!
```

Comment:
This is how we could test our controllers. I think it can be improved but overall looks great and gives us some tests that we don't have now. wdyt @VincentLanglet ?

Reply:
Seems great.

But adding 
```
name: Run Jest
run: npx jest
```
to the github action will lead some issue for other frontend projects like SonataAdmin.

Either
- We add a config for jest (the easy solution, but less dev kit config is better)
- We add jest and some tests to SonataAdmin (is it possible ?)
- We add a way to automatically detect if we need to run jest in the github action (?)

Reply:
We can't add test to SonataAdmin because the javascript is written in a non testable way (or not easily testable), but we can write some fake test to standarize configurations.

To me we can start by just adding npx jest on our main frontend workflow, but this could be also split in the future like we do with php: QA, Test and Build (we don't have build on the php side, but you get the idea).

There is still some work to do on the PHP side and testing side, but we are getting close.

Reply:
This shouldnt be a problem with latest features on dev-kit we can detect packages installed.